godtools/App/Features/Tools/SwiftUI/AllTools/AllToolsListIOS14.swift

```diff
@@ -0,0 +1,76 @@
+//
+//  AllToolsList.swift
+//  godtools
+//
+//  Created by Rachael Skeath on 5/9/22.
+//  Copyright © 2022 Cru. All rights reserved.
+//
+
+import SwiftUI
+
+// MARK: - AllToolsListIOS14
+
+// This view should be visually equivalent to AllToolsList. iOS 14 has no way of removing separators on List, so this is the workaround.
+// TODO: - When we stop supporting iOS 14, remove this view.
+@available(iOS 14.0, *)
+struct AllToolsListIOS14: View {
```

Comment:
Hey @rachaelblue does this fix the separator line issue because its using a ```LazyVStack``` element?

Reply:
Yep.  `LazyVStack` doesn't have any sort of default formatting like `List` does.